src/backend/columnar/sql/citus_columnar--13.2-1--14.0-1.sql

```diff
@@ -1,2 +1,44 @@
 -- citus_columnar--13.2-1--14.0-1
 -- bump version to 14.0-1
+
```

Comment:
I wonder should create older versions of these views in downgrade path as we usually do, or, as we did in some of the similar bugfixes, should those stay as is even after a downgrade?

Reply:
If we keep this view, we’ll only “lose” rows that came from other sessions’ temp columnar tables which nobody should rely on, and which is exactly what bit on PG18.

Reply:
Alternatively, I can add a downgrade script to this PR and open a separate issue to decide later whether to keep or remove it.

What do you think on this @onurctirtir ?